### ISSUE TITLE
Test adding the meta loss to an LSTM model

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#209]

**Comment by soi** [comment]
Should only be done after the additional validation set is set up.

**Comment by soi** [comment]
Also could be tested to add some noise to it to prevent overfitting.
